autoload/neomake/makers/ft/c.vim

```diff
@@ -22,6 +24,26 @@ function! neomake#makers#ft#c#clang()
         \ }
 endfunction
 
+function! neomake#makers#ft#c#clangcheck()
+    if !exists('g:neomake_c_clangcheck_config_file')
+        let g:neomake_c_clangcheck_config_file = '.neomake_clangcheck_config'
+    endif
+
+    return {
+        \ 'exe': 'clang-check',
+        \ 'args': ['%:p', '--', '-fdiagnostic-format=clang']
+                  \ + neomake#utils#ReadConfig(g:neomake_c_clangcheck_config_file),
```

Comment:
That's true, but it's not a problem neomake aims to solve. Other makers don't all have their own config file option in neomake, so this is inconsistent with the rest of the project. I have this in my vimrc as a way to do per-project configurations:

```
let g:project_local_vimrc = findfile('.project.vim', '.;')
if filereadable(g:project_local_vimrc)
    execute 'source ' . g:project_local_vimrc
endif
```